packages/x-data-grid/src/components/virtualization/GridMainContainer.tsx

```diff
@@ -2,7 +2,11 @@ import * as React from 'react';
 import { styled } from '@mui/system';
 import { useGridAriaAttributes } from '../../hooks/utils/useGridAriaAttributes';
 
-const Element = styled('div')({
+const Element = styled('div', {
```

Comment:
Would align with other places (unless we are not following this convention any more)
```suggestion
const GridMainContainerRoot = styled('div', {
```

Reply:
I find that very verbose for no tangible gain, I would like if we could adopt a more relaxed naming convention for internal components. This one never leaves the file.